app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -758,6 +758,7 @@ QueryResult createCirrusQueryResult(List<dynamic> statuses) {
     'searchBuilds': <dynamic>[
       <String, dynamic>{
         'id': '1',
+        'branch': 'master',
```

Comment:
Can we also tests that used a branch different than master? My assumption is that master is hardcoded everywhere and we may miss some places that need changes if we use master.

Reply:
The reason we fix `master` here is that `check for waiting pull requests` applied to `master` only. As suggested, I have added one more test to consider the case with a different branch name, for which we will ignore cirrus statues.